test/models/cheatsheet_test.rb

```diff
@@ -0,0 +1,57 @@
+require 'test_helper'
+
+class CheatsheetTest < ActiveSupport::TestCase
+  def setup
+    @user = users(:janessa)
+    body = %{
+      # Rspec
```

Comment:
But ... there's how to do `rspec` in here? Anyhoo ... for model tests for just the migration, in rspec anyway, you'd do something like ...
```ruby
it { respond_to(:user) }
it { respond_to(:topic) }
# etc.
```

this makes sure that you have everything you think you should have in the database.

Reply:
Haha I just got that from my TIL repo. I should probably be using rspec for testing shouldn't I..

Reply:
As long as you're testing, it doesn't matter that much. I just wanted you to see how it'd look in rspec.